lib/validations/url_create.ts

```diff
@@ -1,28 +1,36 @@
-import { NextRequest } from "next/server";
 import { urlSchema } from "../zod/url";
+import cheerio from 'cheerio';
 
-import { IUrlCreateReq } from "@/interfaces/url";
+const validateURLCreateReq = async (formdata: FormData) => {
+  try {
+    const long_url = formdata.get("longUrl");
+    let title:any = formdata.get("title");
```

Comment:
Why did you use any?

Avoid using any ig here string | null will do the job.

Pls, recheck @VijeshVS